public/js/main.js

```diff
@@ -228,10 +227,23 @@ bingo.controller('bingoController', function($scope, $document, $http, $routePar
         console.log(data);
         $scope.gamecard = data.card.squares;
         $scope.cardid = data.card._id;
-        $scope.roomname = data.game.room;
-        $scope.currentUser = data.user;
-        // $scope.host_name = data.game.host.name;
-        $scope.players = [];
+
+        $scope.roomname = data.roomname;
+        $scope.currentUser = data.currentUser;
+        $scope.host = data.host;
+        $scope.host_name = data.host.name;
+
+        $scope.players = []
+        $scope.players.push($scope.currentUser);
```

Comment:
Please be careful when merging. This undid a change introduced in the [last pull request](https://github.com/RyanEggert/olinjs-bingo/blob/002f2c7ee930b1165387055bd6f74cb599e6e8c6/public/js/main.js#L202-217).
Fortunately, we rewrite over this $scope.players later, so nothing broke. I'll be changing this back in my next pull request.